README.md

```
@@ -24,65 +24,60 @@ To build your app for production:
+||__ _internal.teams.$teamId.members.tsx
+||__ _internal.teams.$teamId.todos.tsx
+||__ _internal.teams.create.tsx
+||__ _internal.teams.index.tsx
+||__ _internal.teams.join.tsx
+||__ index.tsx
 ```
 
-#### Routes Created by Next.js
+#### Routes Created by TanStack Router
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)